lib/utils/HttpHelper.dart

```diff
@@ -91,7 +91,7 @@ class HttpHelper {
 
     request.headers.addAll(
       _buildHeaders(
-        user: user,
+        user: OpenFoodAPIConfiguration.getUser(user),
```

Comment:
You should simplify the code and call `OpenFoodAPIConfiguration.getUser(user)` only once: inside `_buildHeaders`. Does that make sense?

Reply:
Me: Ignore the thing I am doing at the same time for the userAgent, 

Fixed